### ISSUE TITLE
Italic doesn't work

### ISSUE THREAD

**Issue by elliottmangham** [Hube2/acf-medium-editor#21]
Hi,

I have several Medium Editors on my site and all of the options appear to work perfectly apart from _Italic_.

Inspecting it, Italic doesn't appear to wrap the the text in any tag whatsoever, whereas the others do.

Please investigate or advise.

Thank you!

**Comment by elliottmangham** [comment]
Hey @Hube2, is this something you may be able to look into?

Thanks a bunch!

**Comment by Hube2** [comment]
I plan to look into this as soon as I can.

**Comment by elliottmangham** [comment]
Hi @Hube2,

Here's a video example of the issue and the only workaround I can find:
https://youtu.be/WxAMiIfWQZ8

Apply bold, then italics, then disabling bold fixes the issue. But otherwise, the italics option doesn't work alone.

No JavaScript issues found in the console on reload or when formats are selected.
I also disabled all plugins excluding this, ACF and Timber (as it's required) and the issue is still there.

**Specs:**
WordPress: 4.9.4
Advanced Custom Fields Pro: 5.6.10
Chrome: 65.0.3325.181

This has been tested on multiple machines and the same issue is found.

**Comment by Hube2** [comment]
What settings have you included for the field? It could be that some of the options settings for medium editor are causing the issue.

**Comment by elliottmangham** [comment]
Hey @Hube2, did you ever manage to replicate/resolve this?

**Comment by Hube2** [comment]
Yes, I've been looking at this and I'm seeing the same thing in Chrome, but not in other browsers. Honestly, I'm lost as to why, I can't find anything. There have been some minor updates to medium editor since the last time I updated and I'm going to see if updating that fixes the issue here.  

I tried looking through the issues for medium editor to see if this was brought up there, but there are so many issues there and searching them is not getting me any results that help https://github.com/yabwe/medium-editor.

**Comment by elliottmangham** [comment]
Thanks for looking into this @Hube2. Not sure if I can be of any help, though I wish I could!
